cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/BrokerCapacity.java

```diff
@@ -9,20 +9,19 @@ public class BrokerCapacity {
     // CC designates the id of this default broker entry as "-1".
     public static final int DEFAULT_BROKER_ID = -1;
     public static final String DEFAULT_BROKER_DOC = "This is the default capacity. Capacity unit used for disk is in MiB, cpu is in percentage, network throughput is in KiB.";
-
-    public static final String DEFAULT_CPU_UTILIZATION_CAPACITY = "100";  // as a percentage (0-100)
+    public static final String DEFAULT_CPU_CORE_CAPACITY = "1";  // as a percentage (0-100)
```

Comment:
So, is this 1 CPU core? Or is it 100% of available cores? The name suggests to me it is the first one. But the comment suggests the other. It would be great to have it more clear from the name.

Reply:
I think the comment is just a left over of copying the previous line. It should be just 1 core. The comment is wrong.